### PR TITLE
flake: correct nix-darwin flake description

### PR DESCRIPTION
**Pull request by jameschensmith**
### Description

Corrects copy/paste mistake in the nix-darwin flake description.

### Checklist

- [x] Change is backwards compatible.

- [x] Code formatted with `./format`.

- [x] Commit messages are formatted like

    ```
    {component}: {description}

    {long description}
    ```

    See [CONTRIBUTING](https://github.com/nix-community/home-manager/blob/master/docs/contributing.adoc#sec-commit-style) for more information and [recent commit messages](https://github.com/nix-community/home-manager/commits/master) for examples.

### CODE REVIEW

**Review by berbiche — APPROVED** [review]
Thanks!

File: docs/nix-flakes.adoc [hunk 1]
```diff
@@ -188,7 +188,7 @@ is similar to that of NixOS. The `flake.nix` would be:
 [source,nix]
```
Comment (teto) [inline docs/nix-flakes.adoc:188]: shouldn't we include or link templates/nix-darwin/flake.nix here ? else both flakes will go out of sync

Comment (jameschensmith) [reply docs/nix-flakes.adoc:188]: You mean, for example, like `include::../templates/nix-darwin/flake.nix[]`? That's actually a great idea.

Comment (teto) [reply docs/nix-flakes.adoc:188]: yes

**Comment by rycee** [comment]
Thanks! Merged to master now. Unfortunately the include doesn't work since the documentation build happens inside a sandbox that doesn't have access to the wider project code.

**Pull request merged**